py/bitbox02/bitbox02/bitbox02.py

```diff
@@ -269,6 +272,9 @@ def _msg_query(
             print(response)
         return response
 
+    def u2f_query(self, msg: bytes) -> bytes:
```

Comment:
What's the purpose of this? Seems like it ca be deleted as it's u2f, and empty body.

Reply:
ack